Argo/Operators/DecodeDecoded.swift

```diff
@@ -1,43 +1,150 @@
-// MARK: Values
+/**
+  Attempt to decode a value at the specified key into the requested type.
 
-// Pull value from JSON
+  This operator is used to decode a mandatory value from the JSON. If the
+  decoding fails for any reason, this will result in a `Failure` being
+  returned.
+
+  - parameter json: The `JSON` object containing the key
+  - parameter key: The key for the object to decode
+
+  - returns: A `Decoded` value representing the success or failure of the
```

Comment:
In #354 @tonyd256 said a new line should be added here. I'm just saying it here, because he didn't and I'm guessing it's some convention you both overlooked.